### ISSUE TITLE
Does this library support the config option throttle_cb? 

### ISSUE THREAD

**Issue by mtrienis** [confluentinc/confluent-kafka-python#237]
Here is what my setup looks like: 
```
yum list installed | grep librdka
librdkafka1.x86_64 0.9.5_confluent3.2.1-1.el6 @Confluent.dist
pip freeze | grep confluent
confluent-kafka==0.9.4
cat /etc/redhat-release
CentOS release 6.6 (Final)
```
And this is the error when I try to construct the producer?

```
KafkaException: KafkaError{code=_INVALID_ARG,val=-186,str="Property "throttle_cb" must be set through dedicated .._set_..() function"}
```

**Comment by edenhill** [comment]
The throttle callback is not yet exposed in the Python API.

**Comment by mtrienis** [comment]
Thanks, do you think this is a simple extension? If it's trivial enough, I might be able to do a PR.

**Comment by edenhill** [comment]
@mtrienis It should be straight forward, pretty much copying all occurrences of `error_cb` in confluent_kafka/src, tests, etc.

**Comment by rnpridgeon** [comment]
The PR that fixes this has been merged into `adminapi` branch. Once the `adminapi` branch is merged to master it will be available. 

https://github.com/confluentinc/confluent-kafka-python/pull/377
